src/test/java/io/reactivex/internal/operators/observable/ObservableBlockingTest.java

```diff
@@ -209,6 +209,11 @@ public void blockingLastEmpty() {
         Observable.empty().blockingLast();
     }
 
+    @Test(expected = IllegalArgumentException.class)
+    public void blockingIterableNegativeBufferSizeFail() {
```

Comment:
This is covered by the [ParamValidationCheckerTest](https://github.com/ReactiveX/RxJava/blob/2.x/src/test/java/io/reactivex/ParamValidationCheckerTest.java) class which scans all operators for parameter validation.